Rakefile

```diff
@@ -42,9 +42,16 @@ namespace :xcode do
   end
 end
 
-desc 'Lint the Pod'
-task :lint do |task|
-  plain("pod lib lint StencilSwiftKit.podspec --quick", task)
+namespace :lint do
+  desc 'Lint the Pod'
+  task :pod do |task|
+    plain("pod lib lint StencilSwiftKit.podspec --quick", task)
+  end
+  
+  desc 'Lint the code'
+  task :code do |task|
+    plain("PROJECT_DIR=. ./Scripts/swiftlint-code.sh", task)
+  end
```

Comment:
Just a thought: I'm not sure about the readability/logic of grouping those two commands, still undecided.

Because "linting a pod" and "linting the code" seems two different kinds of lints to me. One is checking that the podspec "compiles" (the podspec file doesn't have any syntax error, and that a project with that pod would compile), the other is checking for code style issues.

Maybe it would be more logical to put everything related to SwiftLint in the Rakefile anyway (I mean instead of having separate `.sh` scripts to install swiftlint and lint the code — which were there at first because we didn't have any Rakefile or because we found this install_swiftlint script as is) and have `rake swiftlint:install`, `rake swiftlint:sourcecode` and `rake swiftlint:tests` tasks? (and then concerning the `pod lib lint`, not sure it's worth having a dedicated rake task?)